minio/api.py

```diff
@@ -698,8 +698,7 @@ def fget_object(self, bucket_name, object_name, file_path, metadata=None):
         :param bucket_name: Bucket to read object from.
         :param object_name: Name of the object to read.
         :param file_path: Local file path to save the object.
-        :param metadata: Any additional metadata to be uploaded along
-            with your GET request.
+        :param metadata: Any additional metadata to be added to GET request's headers.
```

Comment:
IMHO we should keep it as request headers,  in put_object it is meant to be saved with the object. Here it is meant to be queried.. so it is better to named request_headers or req_headers makes the code more readable.